### PR TITLE
Firminator patch 2 - Reorder where the DES keys are shown

### PR DESCRIPTION
**Pull request by Firminator**
- Would it be possible to move the keys down to option 4 (Edit DES keys) where you would expect the keys to be shown?
- minor text change
- removed 1 whitespace

### CODE REVIEW

**Comment by pietrushnic** [comment]
@Firminator, thank you. My personal dream would be MrChrombox-like UI: https://github.com/Dasharo/dasharo-issues/issues/225

**Comment by Firminator** [comment]
I think the user interface is pretty awesome so far. It gets the job done. But yes as long time user of Matt's firmware (and his firmware update script) I can see why you consider it the golden standard to compare against.

**Comment by Firminator** [comment]
I made the mistake to PR into 'main'. Can you change it so it goes into Dev branch if you decide to take the commit similar to what ThomaszAIR did with my other PR @ https://github.com/Dasharo/meta-dts/pull/85#issuecomment-1893930362

File: meta-dts-distro/recipes-dts/dts/dts/dts [hunk 1]
```diff
@@ -71,12 +65,17 @@ while : ; do
     echo "  3) Restore firmware from Dasharo HCL report"
   fi
   if [ -n "${DES_IS_LOGGED}" ]; then
-      echo "  4) Edit DES keys"
+      echo "  4) Edit your DES keys"
+      echo "     Logs key    : " ${CLOUDSEND_LOGS_URL}
+      echo "     Download key: " ${CLOUDSEND_DOWNLOAD_URL}
+      echo "     Password    : " ${CLOUDSEND_PASSWORD}
+  fi
```
Comment (TomaszAIR) [inline meta-dts-distro/recipes-dts/dts/dts/dts:72]: @Firminator this `fi` is messing up the script as we later use else to show `Load your DES keys` while DES is not logged. Could you please fix that by just removing it?

**Comment by TomaszAIR** [comment]
Those changes were cherry-picked (https://github.com/Dasharo/meta-dts/commit/1ac80592809eb4f5d4e30b4ab61e1515a409db98 and https://github.com/Dasharo/meta-dts/commit/facde1b188867fc2dcca9f51320cbaf4a5b1b1c6) and released in v1.2.19. @Firminator thank you for the contribution :)

**Pull request closed without merging**